crates/fluvio-cluster/src/start/common.rs

```diff
@@ -45,37 +45,28 @@ pub async fn try_connect_to_sc(
     async fn try_connect_sc(
         fluvio_config: &FluvioConfig,
         expected_version: &Version,
-        timeout: Duration,
     ) -> Result<Fluvio, TryConnectError> {
-        use tokio::select;
-
-        select! {
-            _ = &mut sleep(timeout) => {
-                debug!("timer expired");
-                Err(TryConnectError::Timeout)
-            },
-
-            connection = Fluvio::connect_with_config(fluvio_config) =>  {
-                match connection {
-                    Ok(fluvio) => {
-                        let current_version = fluvio.platform_version();
-                        if current_version == expected_version {
-                            debug!("Got updated SC Version{}", &expected_version);
-                            Ok(fluvio)
-                        } else {
-                            warn!("Current Version {} is not same as expected: {}",current_version,expected_version);
-                            Err(TryConnectError::UnexpectedVersion {
-                                expected: expected_version.clone(),
-                                current: current_version.clone()
-                            })
-                        }
-                    }
-                    Err(err) => {
-                        warn!("couldn't connect: {:#?}", err);
-                        Err(TryConnectError::Unexpected(err))
-                    }
+        match Fluvio::connect_with_config(fluvio_config).await {
+            Ok(fluvio) => {
+                let current_version = fluvio.platform_version();
+                if current_version == expected_version {
+                    debug!("Got updated SC Version{}", &expected_version);
```

Comment:
minot nit, this should follow tracing format where objects are passed rather than format
```
debug!(expected_version,"got updated SC version")
```

Reply:
now it prints:
```
fluvio_cluster::start::common: Got updated SC Version version=0.11.9-dev
```
is that ok?